essentials/function.mdx

```diff
@@ -325,6 +325,119 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
 ```
 It´s currently moderate rain in London with a temperature of 8.0 degrees Celsius.
 ```
+### Example 4: `get_email_resume`
```

Comment:
@fuchsfelipel , what do you think of the idea of ​​function calling having just one example that is already very complete and adding the other examples to a page that only contains examples ? So each example can be better detailed on this examples page

Reply:
Could be on another task